### PR TITLE
fix(DB/Creature): add missing npc loot part 1

### PR DESCRIPTION
**Pull request by Helias**
I've extracted some missing loots from wowhead using this tool [Helias/wowhead-scraper](https://github.com/Helias/wowhead-scraper)
There are around 80 NPC missing loots, I splitted the fixes in 8 PRs.

## Changes Proposed:
add missing NPC loot part 1/8

## SOURCE:
WoWhead uses the "wowhead client" to sniff the data from the retail and it does some statistics, it should be accurate enough as the sniffs usually are.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/missing_npc_loot_1.sql [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+DELETE FROM `creature_loot_template` WHERE (`Entry` = 2396);
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(2396, 4306, 0, 75.0, 0, 1, 0, 2, 2, 'Hans Zandin - Silk Cloth');
+
+-- Fawn (890)
+UPDATE `creature_template` SET `lootid` = 890 WHERE (`entry` = 890);
+DELETE FROM `creature_loot_template` WHERE (`Entry` = 890);
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(890, 2318, 0, 13.33, 0, 1, 0, 1, 1, 'Fawn - Light Leather'),
+(890, 2934, 0, 86.67, 0, 1, 0, 1, 1, 'Fawn - Ruined Leather Scraps');
```
Comment (heyitsbench) [inline data/sql/updates/pending_db_world/missing_npc_loot_1.sql:13]: Hard doubt on this one. Looks strikingly like skinning loot, not creature loot.

Comment (Helias) [reply data/sql/updates/pending_db_world/missing_npc_loot_1.sql:13]: I noticed it, we could remove it if we are not sure

Comment (heyitsbench) [reply data/sql/updates/pending_db_world/missing_npc_loot_1.sql:13]: It and others like it need to be removed. Their source on Wowhead (in the case of 890) lists only 15 samples. This is far too small of a sample size to be legitimately considered IMO.

Comment (Helias) [reply data/sql/updates/pending_db_world/missing_npc_loot_1.sql]: removed loot of 890

Comment (Helias) [reply data/sql/updates/pending_db_world/missing_npc_loot_1.sql]: removed

**Comment by heyitsbench** [comment]
I'll be quite frank: I think these discrepancies should be in an issue and not imported 1:1 in a PR. Things like the Booty Bay Bruiser having loot added I'm fine with, but any critter having loot (especially what should clearly be skinning loot) is very obviously incorrect data on Wowhead's end. I don't think these PRs are a great idea, especially not in their current state.

**Comment by Kitzunu** [comment]
wowhead isnt reliable at all at face value. People use their addon on private servers and shit

**Comment by Nyeriah** [comment]
Technically you "loot" skinning loot so its easy to understand why the data gets compromised

**Comment by Tereneckla** [comment]
16 cases of easily identifiable wrong leather drops in this alone

many of those leather drops are also on critters, so there probably are a few other drops on critters as well

**Comment by Helias** [comment]
I've removed several loots based on your suggetions

**Comment by Tereneckla** [comment]
2396: looted 4 times
2361: looted 6 times (unreliable)
1975: looted 4 times (unreliable)
2392: looted 5 times (unreliable)
2362: looted 4 times (unreliable)
4625: looted 3 times (as a mob in a dungeon people actually do!)
7307: looted 6 times (clearly pickpocket loot that was misplaced by wowhead)
7310: looted 3 times (clearly pickpocket loot that was misplaced by wowhead)
4624: looted 5 times (unreliable)

**Comment by Tereneckla** [comment]
Quick look at the other wowhead imports are:
Lootable critters: no no
Lootable NPCs that transform through script: no no
Very low (sub 10) amount of loot counts
And a few that actually could be legit, but they should use random drops appropriate to their level done through reference

**Comment by Nyeriah** [comment]
Honestly, let's just stop those mass imports. I don't see how adding a few junk items to loot is worth all the possible misplacements and junk data we are getting. Quality > quantity.

If we are missing data, honestly, I believe we should much prefer accuracy than just "get something in place to have something in place", unless its something critical and absolutelly impossible to verify.

**Comment by Helias** [comment]
I agree, we can stop this then.

Anyways, hope that these PRs could help on having a list of potential missing loots

**Pull request closed without merging**